modules/postgresql/metadata.yaml

```diff
@@ -223,7 +223,7 @@ spec:
         connections:
           - source:
               source: github.com/terraform-google-modules/terraform-google-network//modules/vpc
-              version: v9.1.0
+              version: ">= 9.1.0"
```

Comment:
@q2w Let'ts keep an upper bound on the major version as that may contain breaking changes `~> 9.1`